src/main/java/net/pms/formats/SAMI.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
@Sami32 Yeah, discuss it - I guess @valib is the one to discuss it with. It might be best to create an issue for discussing that insted of doing it on a comment here. These comments are so hard to find later.